src/growthbook.ts

```diff
@@ -1,7 +1,12 @@
-import { GrowthBook } from '@growthbook/growthbook-react'
+import { GrowthBook } from '@growthbook/growthbook'
 import * as RudderAnalytics from 'rudder-sdk-js'
 
 export type GrowthBookType = GrowthBook
```

Comment:
```suggestion
export type GrowthBookType = GrowthBook
```

😕 Can we not do this, and just use `GrowthBook` directly?

Reply:
Yes, I thought we can use export this type for instances but I hope it'll rare case